### ISSUE TITLE
8 player support for zombies?

### ISSUE THREAD

**Issue by Jbleezy** [Nukem9/LinkerMod#24]
Do you guys hink it would be possible to add 8 player support for BO1 Zombies? This would be amazing.

**Comment by Nukem9** [comment]
It's probably possible, but I have no way of testing it reliably.

**Comment by Jbleezy** [comment]
I have a few people who would be willing to test. Could we try getting it working?

**Comment by Nukem9** [comment]
Yeah I'll have to look into it first. I'm not positive what is hardcoded to 4 players or not, so I'll update this if I find anything.

**Comment by Jbleezy** [comment]
Sounds great, thank you!

**Comment by Wijaxu** [comment]
@Nukem9 you'll have to rewrite the files DemonWare

**Comment by Jbleezy** [comment]
Did you find anything?

**Comment by Nukem9** [comment]
Right now it "works" when increasing com_maxclients to 8. It's still untested since it's 1 slot used/7 slots free...some bot clients will be needed to really check.

As for party lobbies: haven't checked at all.

**Comment by Jbleezy** [comment]
Or we could test with 5+ real players.
But I'm guessing that would require the party lobbies change too.
Can you look into that?

**Comment by Nukem9** [comment]
So I checked with a few bot clients and this isn't going to happen any time soon. `clientInfo_t` (bgs_t), `clientState_s`, `snapshot_s`, `clientInfo_t`, and `PartyState_s` need to be patched. That's getting to a few hundred patches to fix up all related functions/structs.

**Comment by Jbleezy** [comment]
Is there a way I could help out in getting it to work?

**Comment by Jbleezy** [comment]
Any way I could help?

**Comment by SE2Dev** [comment]
`Any way I could help?`
Unless you know how to, and are willing to reverse engineer and rewrite several hundred functions, there really isn't any way that you can help. Essentially any place in the game that those structs (mentioned above by @Nukem9) are used, needs to be entirely rewritten, the amount of time and effort that this would take would be (at the very least) comparable to the current amount of code for game_mod. This really isn't feasible at the moment.
